src/Module.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LotGD\Core;
+
+/**
+ * Classes which provide module functionality should implement this interface.
+ */
+interface Module extends EventHandler
```

Comment:
All events will come through one method, `handleEvent`, yes. Users can `switch` on that event and do what they want. Most users won't need more than one class I suspect, but if they do, they can distribute the events to different classes.